packages/crayons-core/src/components/checkbox/checkbox.tsx

```diff
@@ -105,22 +114,25 @@ export class Checkbox {
         role='checkbox'
         tabIndex='0'
         aria-disabled={this.disabled ? 'true' : 'false'}
-        aria-checked={this.checked}
+        aria-checked={`${this.checked}`}
+        aria-label={`${this.label}`}
+        aria-describedby="description"
         onFocus={() => this.onFocus()}
         onBlur={() => this.onBlur()}
-      >
-        <input type='checkbox' ref={(el) => (this.checkbox = el)}></input>
-        <label>
-          <span class='text'>
-            <slot />
-          </span>
-          <br />
-          {this.label !== '' ? (
-            <span class='label-field'>{this.label}</span>
-          ) : (
-            ''
-          )}
-        </label>
+        >
```

Comment:
Fix indentation here. There seem to be 2 extra spaces.

Reply:
Accommodated in latest commit